custom_benches/src/main.rs

```diff
@@ -0,0 +1,114 @@
+//! Translated from rapier benchmark.
+//!
+//! <https://github.com/dimforge/rapier/blob/87ada34008f4a1a313ccf8c3040040bab4f10e69/benchmarks3d/many_pyramids3.rs>
+
+use benches_common::default_app;
+use benches_common::wait_app_start;
+use bevy::prelude::*;
+use bevy_rapier3d::dynamics::RigidBody;
+use bevy_rapier3d::geometry::Collider;
+use bevy_rapier3d::math::Vect;
+use bevy_rapier3d::plugin::RapierContext;
+
+pub fn create_pyramid(commands: &mut Commands, offset: Vect, stack_height: usize, rad: f32) {
```

Comment:
I'm not a fan of code duplication with `bevy_rapier3d/benches` ; but I'm not sure how to solve it.

Reply:
I added shared setup code to the custom benchmark `bevy_rapier_benches3d`